content/docs/introduction/roadmap.md

```diff
@@ -86,12 +86,16 @@ GitHub issue: [#398](https://github.com/prometheus/prometheus/issues/398)
 
 **Server-side metric metadata support**
 
-At this time, metric types and other metadata are only used in the client
-libaries and in the exposition format, but not persisted or utilized in the
-Prometheus server. We plan on making use of this metadata in the future. For
-example, we could suggest automatic rates over counters, warn users if they
-take the rate of a gauge, or display metric documentation strings. The details
-of this are still to be determined.
+At this time, metric types and other metadata are only used in the
```

Comment:
We might want to standardize on a line length for wrapping to cause less diff noise if the only change to a paragraph is an addition at the end. (e.g. I use the vim default of 79 chars plus newline)